### PR TITLE
px4flow.md update

### PR DESCRIPTION
**Pull request by szobov**

### CODE REVIEW

**Comment by szobov** [comment]
@hamishwillee ping

**Comment by hamishwillee** [comment]
Thanks - I did miss this. What specific evidence are you using that it doesn't work on 1.8.2 and that it does on 1.9.x (did you test on master and it worked?). Is it certain this won't be fixed on 1.8.2?

**Comment by szobov** [comment]
> did you test on master and it worked?

I've checked it on master with mentioned hardware and, yes, it works.
But I checked it about two weeks ago, maybe something was changed.
If you want I can check it with fresh master or I can wait till the release date.

> Is it certain this won't be fixed on 1.8.2?

In px4 slack @dagar said:
> 2. No changes will be made in the previous stable branch.
<sub>[View in Slack](https://px4.slack.com/archives/C0W2KUFFT/p1553091076126900)</sub>

So, I can't be absolutely sure, that it's a truth, but I don't know how I can prove it, sorry.

File: en/sensor/px4flow.md [hunk 1]
```diff
@@ -341,7 +341,7 @@ The PX4FLOW module outputs [MAVLink](https://mavlink.io/en/) packets on USB and
 
 The following list gives a short explanation of the current available parameters in the PX4FLOW firmware.
 
-> **Caution** Parameters are currently not written to ROM (reset at power loss).
+> **Caution** Parameters are currently not written to ROM (reset at power loss). But if you want to change them you can build your own version of firmware using instruction `PX4FLOW Developer Guide` mentioned  above. Code with parameters you can find [here](https://github.com/PX4/Flow/blob/master/src/modules/flow/settings.c).
```
Comment (hamishwillee) [inline en/sensor/px4flow.md]: @szobov Which document is the `PX4FLOW Developer Guide` referenced?

**Comment by hamishwillee** [comment]
Thanks @szobov . I have merged this and will slightly tidy as a post process. If you could verify the v1.9 again in the coming weeks that would be great.

File: en/sensor/px4flow.md (hunk 1 above)
Comment (szobov) [reply en/sensor/px4flow.md]: Actually, I don't know how to make self references inside one document, so it's this document: https://github.com/szobov/px4_user_guide/blob/c6671b794ffd1935ceccb3a6b7262d0c2f5c9516/en/sensor/px4flow.md#px4flow-developer-guide-developer_guide

Comment (hamishwillee) [reply en/sensor/px4flow.md]: Thanks very much. FYI only, you can link to any anchor in the doc using the syntax like `[text displayed for the link](#anchor-name)`. The anchor name can come from the heading as one is created automatically (and you can copy from the table of contents) or you can define what you want the anchor to be - e.g. if you have a heading defined you can use curly braces to define the anchor: `## Some heading {#your_own_anchor}`

I've cross linked now.
https://github.com/PX4/px4_user_guide/commit/81fe8ec3f3e6059be355ac128ec8fd830d328fc0

**Pull request merged**